lib/workos/sso/sso.ex

```diff
@@ -94,7 +94,7 @@ defmodule WorkOS.SSO do
     access_token = %{
```

Comment:
`access_token` above is a map, should not be concatenated to a keyword list. It is producing a wrong result
```
iex> [foo: "bar"] ++ %{access_token: 123}
[{:foo, "bar"} | %{access_token: 123}]
```

The section can be rewritten as
```
opts = opts ++ [access_token: access_token]
```
or simply
```
opts = opts |> Keyword.put(:access_token, access_token)
```

@jacobiajohnson @blairlunceford